kolibri/utils/server.py

```diff
@@ -86,22 +85,29 @@ def __init__(self, bus, port):
         self.workers = None
 
     def start(self):
-        # Initialize the iceqube scheduler to handle scheduled tasks
-        scheduler.clear_scheduler()
-
+        # If pinging is not disabled by the environment
         if not conf.OPTIONS["Deployment"]["DISABLE_PING"]:
 
-            # schedule the pingback job
-            from kolibri.core.analytics.utils import schedule_ping
+            # schedule the pingback job if not already scheduled
```

Comment:
I think I'd prefer to put this check inside the `schedule_ping` function rather than in here.

Reply:
Yes, I'll do that.